functions.php

```diff
@@ -46,10 +46,36 @@ function insertToDB(object $db, string $title, string $author, string $release_y
  */
 function softDelete(object $db, string $id): void
 {
-    $query = $db->prepare("UPDATE `books` SET `deleted` = '1' WHERE `id` = :id;");
+    $query = $db->prepare('UPDATE `books` SET `deleted` = \'1\' WHERE `id` = :id;');
     $query->execute(['id' => $id]);
 }
 
+/**
+ * @param string $column - desired column in database table, decided by form on index.php
+ * @param string $value - desired new value for the above specified column
+ * @param $id - the database id of the record to be amended
+ *
+ * @return array - contains [0]-> an UPDATE SQL query, [1]-> $value, [2]-> $id
+ */
+function createUpdateQuery(string $column, string $value, string $id):array
+{
+    $column = trim(htmlspecialchars($column));
+    $result[] = 'UPDATE `books` SET `' . $column . '` = :val WHERE `id` = :id;';
+    $result[] = trim(htmlspecialchars($value));
+    $result[] = trim(htmlspecialchars($id));
```

Comment:
Suggest giving these items keys rather than going indexed, makes it easier to tell whats going on when the results get used in updateDBItem